runtime/near-vm-runner/src/tests/runtime_errors.rs

```diff
@@ -54,11 +54,7 @@ fn test_simple_contract() {
 #[test]
 fn test_multiple_memories() {
     test_builder()
-        .wasm(&[
-            0, 97, 115, 109, 1, 0, 0, 0, 2, 12, 1, 3, 101, 110, 118, 0, 2, 1, 239, 1, 248, 1, 4, 6,
-            1, 112, 0, 143, 129, 32, 7, 12, 1, 8, 0, 17, 17, 17, 17, 17, 17, 2, 2, 0,
-        ])
-        // Wasmtime classifies this error as link error at the moment.
+        .wasm(b"\x00\x61\x73\x6d\x01\x00\x00\x00\x05\x07\x02\x01\x01\x02\x01\x03\x04")
```

Comment:
Nit: I know this is pre-existing, but ideally we could annotate what parts of this bytestring mean. For `wast` `(module binary)` tests, I would usually approach this as such:

```wast
(module binary 
    "\x00\x61\x73\x6d"  ;; wasm header
    "\x01\x00\x00\x00"  ;; version 1
    "..."               ;; ...
)
```